myhdl/conversion/_toVHDL.py

```diff
@@ -113,6 +113,7 @@ class _ToVHDLConvertor(object):
                  "use_clauses",
                  "architecture",
                  "std_logic_ports",
+                 "no_initial_value"
```

Comment:
I mean, get rid of the 'no' since it is counter intuitive. I suggested plural for consistency.